documentation/src/main/wiki/StatusCodesDataFilters.md

```diff
@@ -1,5 +1,7 @@
 #### Status Codes Data Filters
 
+Data filters will be apply only for codes contained in the `filterange`. If the `filterRange` isn't provided, default range will be used.
```

Comment:
Please fix the `filterange` typo. 
What about codes contained in the `filterCodes` parameter - will data filters be applied for them?

Reply:
Updated